src/client/pages/OfferNew/Perils/PerilModal.tsx

```diff
@@ -19,235 +21,116 @@ interface PerilModalProps {
-  left: 0;
-  background: linear-gradient(
-    to right,
-    ${colorsV3.white} 30%,
-    ${hexToRgba(colorsV3.white, 0)} 100%
-  );
-`
-
-const RightGradient = styled(Gradient)`
-  right: 0;
-  justify-content: flex-end;
-  background: linear-gradient(
-    to left,
-    ${colorsV3.white} 30%,
-    ${hexToRgba(colorsV3.white, 0)} 100%
-  );
-`
-
-const DirectionButton = styled('button')`
-  width: 100%;
-  height: 100%;
-  display: flex;
+  flex-shrink: 0;
+  width: 2rem;
+  height: 2rem;
+  padding: 0;
   cursor: pointer;
   background: none;
-  border: none;
+  border: 1px solid currentColor;
+  border-radius: 0.5rem;
+
   svg {
     transition: all 0.15s ease-in-out;
-    fill: ${colorsV3.gray500};
+    fill: ${colorsV3.gray900};
   }
   :focus {
     outline: none;
   }
-  &:hover {
-    svg {
-      fill: ${colorsV3.gray900};
-    }
-  }
 `
 
-const BackButton = styled(DirectionButton)`
-  margin-left: 0.625rem;
-  justify-content: flex-start;
-`
-
-const ForwardButton = styled(DirectionButton)`
-  margin-right: 0.625rem;
-  justify-content: flex-end;
-`
-
-const Content = styled('div')`
-  padding: 3.5rem;
-
-  @media (max-width: 450px) {
-    padding: 2rem;
-  }
+const Content = styled.div`
+  display: block;
 `
 
 const Title = styled.div`
-  font-family: ${fonts.HEDVIG_LETTERS_STANDARD};
-  font-size: 2.5rem;
+  margin-left: 0.5rem;
+  margin-right: 0.5rem;
+  font-size: 1.5rem;
   line-height: 1.2;
+  text-align: center;
   color: ${colorsV3.gray900};
-  margin-bottom: 1rem;
-  letter-spacing: -0.01em;
 
-  @media (max-width: 600px) {
-    font-size: 1.5rem;
-    margin-bottom: 0.5rem;
+  ${MEDIUM_SMALL_SCREEN_MEDIA_QUERY} {
+    font-size: 2rem;
   }
 `
 
 const Description = styled.div`
-  font-size: 1.125rem;
-  line-height: 1.5;
+  margin-bottom: 3rem;
+  font-size: 1rem;
+  line-height: 1.33;
   color: ${colorsV3.gray900};
-`
-
-const CoverageWrapper = styled.div`
-  margin-top: 2.5rem;
-  display: flex;
-  flex-direction: row;
-  width: 100%;
 
   @media (max-width: 600px) {
-    flex-direction: column;
+    font-size: 1.125rem;
   }
 `
 
-const CoverageList = styled.div`
-  width: 50%;
-
-  :first-of-type {
-    margin-right: 1rem;
+const CoverageWrapper = styled.div`
+  @media (min-width: 600px) {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
   }
 
-  :last-child {
-    margin-left: 1rem;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    flex-direction: column;
   }
+`
 
-  @media (max-width: 600px) {
-    width: 100%;
-
-    :first-of-type {
-      margin: 0 0 2.5rem 0;
-    }
-
-    :last-child {
-      margin: 0;
-    }
-  }
+const CoverageList = styled.div`
+  width: 100%;
+  margin-bottom: 2rem;
 `
 
 const CoverageListTitle = styled.div`
-  font-size: 1rem;
-  line-height: 1.5;
+  font-size: 1.5rem;
   color: ${colorsV3.gray900};
-  font-weight: 600;
   margin-bottom: 0.75rem;
+  padding-bottom: 0.75rem;
+  border-bottom: 1px solid ${colorsV3.gray900};
 `
 
 const CoverageListItem = styled.div`
```

Comment:
Should this perhaps be a `li` and then we wrap the list in an `ul` element? That makes it more semantically correct at least. But perhaps a nit

Reply:
Great idea! Will do that :)